src/compiler/parsing/watf/grammar.js

```diff
@@ -2,9 +2,15 @@
 
 const {tokens, keywords} = require('./tokenizer');
 const t = require('../../AST');
+const {codeFrameColumns} = require('@babel/code-frame');
```

Comment:
This should probably be a prod dep, I'm not sure yet if we should include it in prod.